.env.development

```diff
@@ -1,3 +1,4 @@
+AWS_DEFAULT_REGION=eu-west-2
```

Comment:
Our gems depend on this value being set so this creates a convention that reduces the configuration a dev needs to do